Plugins/GitUIPluginInterfaces/IGitUICommands.cs

```diff
@@ -156,5 +156,7 @@ public interface IGitUICommands
         bool StartVerifyDatabaseDialog();
         bool StartViewPatchDialog();
         bool StartSparseWorkingCopyDialog();
+        void AddCommitTemplate(string key, Func<string> addingText);
+        void RemoveCommitTemplate(string key);
```

Comment:
I suppose, I have no permission to use `CommitTemplateManager` class in plig-in, I have access only to `IGitUICommands` from `GitUIPluginInterfaces` assembly